scripts/build_zip.sh

```diff
@@ -22,10 +22,23 @@ EOF
   exit 1
 fi
 
+build_head_option=
```

Comment:
Two different option sets for building from head versus building release candidates